addon/services/ember-oauth2.js

```diff
@@ -81,10 +81,13 @@ export default Ember.Service.extend(Ember.Evented, {
                     On reject returns Object with reference to dialog and error.
    */
   openWindow(url) {
-    let dialog = window.open(url, 'Authorize', 'height=600, width=450');
-    if (window.focus && dialog) {
-      dialog.focus();
-    }
+    let windowWidth = 600;
+    let windowHeight = 800;
```

Comment:
maybe there is a better way to set the *heigh* and *width* rather than hard coding here. Either pass in an optional param to `openWindow` or make a developer module level `const`

Reply:
>  make a developer module level `const`

I wasn't sure what `developer module level const` meant to I made a change to allow `width` and `height` to be set by optional parameters. I did keep `width=600` and `height=800` as defaults -- they seem to be reasonable defaults for desktop. Let me know what you think. Happy to explore your other alternative too if you could elaborate!